scripts/testnet/include/config.sh

```diff
@@ -205,3 +205,19 @@ updateJSONValue() {
 
   sed -i "s,\"$key\": .*\$,\"$key\": $escaped_value\,," $filename
 }
+
+changeConfigForHardfork(){
+  pushd $TESTNETDIR/node/config
+
+  export FIRST_PUBKEY=$(cat nodesSetup.json | grep pubkey -m 1 | cut -c18-209)
```

Comment:
```suggestion
  export FIRST_PUBKEY=$(cat nodesSetup.json | grep pubkey -m 1 | sed -E 's/^.*"([0-9a-f]+)".*$/\1/g')
```
`cut` is too fragile, depends on exact character counts. The above `sed` should extract the hexadecimal number from the `grep`ped line independently of tabs / spaces / quotes, commas etc.

Reply:
Done, please check.